### PR TITLE
nginx create

### PR DESCRIPTION
**Pull request by KnyazkovE**
скрипт + конфиг

### CODE REVIEW

**Comment by er2805620** [comment]
sudo cp ./web-server.conf /etc/nginx/conf.d/

not good place
https://docs.nginx.com/nginx/admin-guide/basic-functionality/managing-configuration-files/
conf.d/
set of feature‑specific files stored in the /etc/nginx/conf.d directory and use the [include](https://nginx.org/en/docs/ngx_core_module.html#include) directive in the main nginx.conf file to reference the contents of the feature‑specific files.


So
```
sudo cp ./web-server.conf /etc/nginx/sites-available/
sudo ln -s /etc/nginx/sites-available/web-server.conf /etc/nginx/sites-enabled/
```

File: setup/nginx-web/nginx-install-ubuntu.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+sudo apt install -y curl gnupg2 ca-certificates lsb-release ubuntu-keyring
+curl https://nginx.org/keys/nginx_signing.key | gpg --dearmor \
```
Comment (the2pizza) [inline setup/nginx-web/nginx-install-ubuntu.sh:3]: Currently the script is not idempotent. (If i run it second time It fails) Nice to have the ability to run it multiple times without side effects

**Review by the2pizza — CHANGES_REQUESTED** [review]
Please take a look at my comment about the script

**Comment by the2pizza** [comment]
done - https://github.com/nezavisimost/FuckRKN1/pull/83

**Pull request closed without merging**